nsv13/game/lazylift.dm

```diff
@@ -0,0 +1,394 @@
+Lifts need one or more doors to be placed. Otherwise it will forcibly make one for you in a random location. Lifts require doors to function. That's why we use indestructible turbolift airlocks as standard. However, that's not to stop you from using a regular ol' door.
+Turbolifts need to be in a unique turbolift area.
+Voice activation mode picks up areas based off of their in-world position. If you re-use areas over multiple decks, expect issues (You really shouldn't be doing that anyway...).
+Example:
+
+[]LIFT[]
+[]	  []
+[]DOOR[]
+
+For a simple lift.
+
+Things of note:
+You're fine to place your own turbolift doors down. If the lift can't locate any, it'll place one of its own out of spite.
+
+
+
+That's it, ok bye!
+
+*/
+
+//Mappers, DON'T USE ME! Use the other one.
+
+/area/shuttle/turbolift
+	ambientsounds = list()
+	looping_ambience = 'nsv13/sound/effects/lift/elevatormusic.ogg' //Mandatory.
+
+/obj/machinery/lazylift_button
+	name = "Turbolift call button"
+	desc = "A button that can call a turbolift to your location, so that you can board it. Be sure to mash it as often as physically possible."
+	icon = 'icons/obj/turbolift.dmi'
+	icon_state = "button"
+	can_be_unanchored = FALSE
+	density = FALSE
+	anchored = TRUE
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF | FREEZE_PROOF
+	verb_say = "beeps"
+	verb_ask = "beeps"
+	verb_exclaim = "beeps"
+	use_power = IDLE_POWER_USE
+	idle_power_usage = 2
+	var/obj/machinery/lazylift/lift = null //Pointer to the lift controller in our area on this deck, we then ask its master to path the lift to us, assuming it isn't busy or anything.
+
+/obj/machinery/lazylift_button/attack_robot(mob/user)
+	return attack_hand(user)
+
+/obj/machinery/lazylift_button/attack_ai(mob/user)
+	return attack_hand(user)
+
+/obj/machinery/lazylift_button/attack_hand(mob/user)
+	if (stat & NOPOWER || !lift)
+		to_chat(user, "<span class='warning'>Nothing happens...</span>")
+		return
+	if(isliving(user) && user.mouse_opacity) //Don't display things like AIs or admin ghosts clicking the button.
+		visible_message("<span class='notice'>[user] [pick("calmly presses", "mashes", "pokes", "slaps", "hits")] [src] </span>", "<span class='notice'>You push [src].</span>")
+	flick("button_lit", src)
+	lift.master.path_to(lift.deck, lift.master.platform_location.deck)
+
+/obj/machinery/door/airlock/ship/public/glass/turbolift
+	name = "turbolift door"
+	desc = "A bulkhead which opens and closes."
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF | FREEZE_PROOF //Go away greytiders
+
+//Lifts speak a lotta languages, alien isn't one of 'em.
+/datum/language_holder/synthetic/turbolift
+	languages = list(/datum/language/common)
+	shadow_languages = list(/datum/language/common, /datum/language/machine, /datum/language/draconic, /datum/language/drone)
+
+/obj/machinery/lazylift
```

Comment:
EW override what you need to on the bee version by being a child there.

Reply:
Yeah unfortunately these two systems are totally incompatible, so extension wouldn't help here.